### PR TITLE
ubuntu20.sh

### PR DESCRIPTION
**Pull request by VISHAWANATH953**
Making add on this many times am runs vncserver and exit on this it will runs also exited but runs vncserver so this added on this thats can exit all this background running processes so this matter for i am added on this

### CODE REVIEW

**Comment by VISHAWANATH953** [comment]
If this was ok for you make change all script thats run a linux binarys

**Review by deril1 — CHANGES_REQUESTED** [review]
VISHAWANATH953:patch-1

**Review by deril1 — CHANGES_REQUESTED** [review]
Installer/Ubuntu20/ubuntu20.sh

**Review by jmodes — APPROVED** [review]

**Review by jonbrown1981 — APPROVED** [review]

**Review by jonbrown1981 — APPROVED** [review]
`@`

**Review by jonbrown1981 — APPROVED** [review]

File: Installer/Ubuntu20/ubuntu20.sh [hunk 1]
```diff
@@ -41,6 +41,7 @@ cd \$(dirname \$0)
 unset LD_PRELOAD
 command="proot"
 command+=" --link2symlink"
+command+=" --kill-on-exit"
```
Comment (daddivan69) [inline Installer/Ubuntu20/ubuntu20.sh:44]:
```suggestion
command+=" --kill-on-exit"
```

**Review by daddivan69 — APPROVED** [review]

**Review by daddivan69 — APPROVED** [review]

**Review by jmodes — APPROVED** [review]
Works

**Comment by jmodes** [comment]
??

File: Installer/Ubuntu20/ubuntu20.sh (hunk 1 above)
Comment (VISHAWANATH953) [inline Installer/Ubuntu20/ubuntu20.sh:44]: ok, this was seen like double things like it.

**Comment by ultrahacx** [comment]
Thanks for your effort but we'll be adding the following changes with some additional binds so Im closing this pull request. We still appreciate the effort to make the community better♥.

**Pull request closed without merging**